README.md

```diff
@@ -449,6 +467,32 @@ export default MyComponent = () => {
 
 ## Migrating between versions
 
+### Migrating from 4.x to 5.x
+
+From version 5.0, to remove coupling with `react-router`, you need to pass `navigate` prop to `Security`.
```

Comment:
I would skip the why here and just focus on what they need to do (the second half of the sentence).